blazor/Pages/Index.razor

```diff
@@ -0,0 +1,2 @@
+@page "/"
+<div>Hello World!</div>
```

Comment:
Move this markup (without the page directive) into App.Razor and remove the Pages directory.